pkg/gofr/datasource/surrealdb/utils.go

```diff
@@ -0,0 +1,20 @@
+package surrealdb
+
+import (
+	"regexp"
+	"strings"
+)
+
+// Clean takes a string query as input and performs two operations to clean it up:
+// 1. It replaces multiple consecutive whitespace characters with a single space.
+// 2. It trims leading and trailing whitespace from the string.
+// The cleaned-up query string is then returned.
+func Clean(query string) string {
```

Comment:
I don't think there is a need to make it exported


```suggestion
func clean(query string) string {
```